src/test/java/org/jenkins/plugins/lockableresources/LockableResourceRootActionSEC1361Test.java

```diff
@@ -80,8 +80,8 @@ private void checkXssWithResourceName(String resourceName) throws Exception {
 
         HtmlElement reserveButton = null;
         for (HtmlElement b : allButtons) {
-            String onClick = b.getAttribute("onClick");
-            if (onClick != null && onClick.contains("reserve")) {
+            String action = b.getAttribute("data-action");
+            if (action != null && action.contains("reserve")) {
```

Comment:
I'm not sure if this test makes enough sense to keep it around at this point. The code for which it was originally written has been changed to the point that it's barely recognizable.

Reply:
A valid point; on the other hand, it does provide some automated coverage of the frontend, which is arguably valuable in and of itself…